module/core/diagnostics_tools/tests/inc/layout_test.rs

```diff
@@ -70,10 +70,24 @@ tests_impls!
 fn cta_trybuild_tests()
 {
   let t = test_tools::compiletime::TestCases::new();
-  t.compile_fail( "tests/inc/snipet/cta_type_same_size_fail.rs" );
-  t.compile_fail( "tests/inc/snipet/cta_type_same_align_fail.rs" );
-  t.compile_fail( "tests/inc/snipet/cta_ptr_same_size_fail.rs" );
-  t.compile_fail( "tests/inc/snipet/cta_mem_same_size_fail.rs" );
+
+  let current_exe_path = std::env::current_exe().expect( "No such file or directory" );
```

Comment:
what for?

Reply:
Because here was an relative path to file with tests and that's why try_build tests failed in other crates